apps/trading/pages/_app.page.tsx

```diff
@@ -124,12 +125,14 @@ function VegaTradingApp(props: AppProps) {
   }
 
   return (
-    <HashRouter>
-      <Bootstrapper>
-        <AppBody {...props} />
-      </Bootstrapper>
-      <NodeSwitcherDialog open={nodeSwitcherOpen} setOpen={setNodeSwitcher} />
-    </HashRouter>
+    <Suspense fallback={<AppLoader />}>
```

Comment:
Not sure why you've added Suspense here. It doesn't do anything because we aren't acutally 'suspending' anything in our children

Reply:
Oh my mistake, the useTranslation hook triggers it